cpp/switch.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
```

Comment:
// Contribution list modification not allowed but since it is inside cpp folder, approved.